apps/web/src/server/app/validation/validation.controller.js

```diff
@@ -1,13 +1,52 @@
 import { findAllNewAppeals } from './validation.service.js';
 
+/**
+ * Given an item object transform it into - govuk/components/table/macro.njk" import govukTable format
+ *
+ * @param {*} item to do
+ * @returns {any} to do
+ */
+function makeGovTableRow(item){
+	return [
+		{ html: '<a href="#">'  + item.AppealReference + '</a>' },
```

Comment:
Please don't build html in the server code. It should be done in the NJK files.

Reply:
I am surprised at this as I thought this is exactly what we had agreed to do. It surprised me at the time. But will move this out.

Reply:
Perhaps something was lost in the comms and I apologies if that is the case but I would have never said to put html in the server code.

Reply:
moved into main function